### PR TITLE
Implemented Dashboard UI, added translations, routes and colors as per custom theme

### PR DESCRIPTION
**Pull request by Rushikesh-Sonawane99**
## Summary by CodeRabbit

- **New Features**
	- Introduced a dashboard for managing student attendance and classes with interactive navigation.
	- Enhanced the Header component with improved language selection and added visual dividers.
- **Style**
	- Updated the theme with a new color shade and adjusted the background color for a specific style.
	- Implemented theme-based dynamic styling for text and border colors in multiple components.
- **Refactor**
	- Simplified the `Header` component structure for better maintainability.
	- Adopted theme-based styling in several components for consistency and easier theme management.
- **Chores**
	- Added a new route for the dashboard in the application's routing system.

### CODE REVIEW

File: src/components/CohortCard.tsx [hunk 1]
```diff
@@ -20,7 +20,8 @@ const Root = styled('div')(({ }) => ({
     height: 56,
     border: '1px solid #EDE1CF',
```
Comment (itsvick) [inline src/components/CohortCard.tsx:21]: @Rushikesh-Sonawane99 use theme colors

File: src/pages/Dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+import { useTheme } from '@mui/material/styles';
+
+interface DashboardProps {
+  //   buttonText: string;
+}
+
+const Dashboard: React.FC<DashboardProps> = () => {
+  const { t } = useTranslation();
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
```
Comment (itsvick) [inline src/pages/Dashboard.tsx:35]: @Rushikesh-Sonawane99 why there are two function to handle open and close, instead use `toggleModal`

File: src/pages/Dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+  const handleClose = () => setOpen(false);
+  const [classes, setClasses] = React.useState("");
+  const theme = useTheme<any>();
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setClasses(event.target.value as string);
+  };
+  const style = {
+    position: "absolute" as "absolute",
```
Comment (itsvick) [inline src/pages/Dashboard.tsx:44]: @Rushikesh-Sonawane99 remove `as "absolute"`

File: src/pages/Dashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+    top: "50%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    width: 300,
+    bgcolor: "background.paper",
```
Comment (itsvick) [inline src/pages/Dashboard.tsx:49]: @Rushikesh-Sonawane99 use theme color

File: src/pages/Dashboard.tsx [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+                  <Typography variant="h6" component="h2">
+                    25 May 2024
+                  </Typography>
+                </Box>
+                <Box onClick={() => handleClose()}>
+                  <CloseIcon />
+                </Box>
+              </Box>
+              <Divider sx={{ borderBottomWidth: "0.2rem" }} />
+              <Typography sx={{ mt: 2 }}>
```
Comment (itsvick) [inline src/pages/Dashboard.tsx:135]: @Rushikesh-Sonawane99 dont use box inside typography
Typography meant for text content only

Comment (Rushikesh-Sonawane99) [reply src/pages/Dashboard.tsx:135]: > @Rushikesh-Sonawane99 dont use box inside typography Typography meant for text content only

Removed typography

File: src/pages/Dashboard.tsx (hunk 3 above)
Comment (Rushikesh-Sonawane99) [reply src/pages/Dashboard.tsx:44]: > @Rushikesh-Sonawane99 remove `as "absolute"`

Removed as absolute

File: src/pages/Dashboard.tsx (hunk 2 above)
Comment (Rushikesh-Sonawane99) [reply src/pages/Dashboard.tsx:35]: > @Rushikesh-Sonawane99 why there are two function to handle open and close, instead use `toggleModal`

Handled toggle with one function

File: src/pages/Dashboard.tsx (hunk 4 above)
Comment (Rushikesh-Sonawane99) [reply src/pages/Dashboard.tsx:49]: > @Rushikesh-Sonawane99 use theme color

added custom theme color

**Pull request merged**